src/app/features/course-info/course-info.component.ts

```diff
@@ -4,64 +4,64 @@ import { ButtonLabels } from "@app/shared/constants/button-labels";
 import { Location } from "@angular/common";
 import { CoursesStoreService } from "@app/services/courses-store.service";
 import { Course } from "@app/shared/models/course.model";
-import { forkJoin, Subscription } from "rxjs";
+import { map, Observable, Subscription } from "rxjs";
+import { CoursesStateFacade } from "@app/store/courses/courses.facade";
 
 @Component({
   selector: "app-course-info",
   templateUrl: "./course-info.component.html",
   styleUrls: ["./course-info.component.scss"],
 })
 export class CourseInfoComponent implements OnInit, OnDestroy {
-  course!: Course | undefined;
+  course!: Course | null;
```

Comment:
What purpose does this class member serve with having a course observable on the next line?